### PR TITLE
fix(gradle): refactored pomPath to toolConfigurationPath and fixed up…

### PR DESCRIPTION
**Pull request by hanna-modica**
… documentation for gradle and cli

-fix
-- refactored pomPath in plugin configuration for gradle to toolConfigurationPath 

- doc improvement
-- removed mavenLocal() and replaced with jcenter() in dependencies (gradle)
-- renamed TinaConfiguration to AntennaConfiguration (gradle)
-- mentioned hyphens can't be used in properties (gradle & cli)
-- renamed pom.xml to toolConfiguration.xml (gradle & cli)
-- renamed POM_FILE to TOOL_CONF_FILE in example for script (cli)

Signed-off-by: Hanna Modica <Hanna.Modica@bosch-si.com>

### CODE REVIEW

**Comment by neubs-bsi** [comment]
@hanna-modica merge conflicts with master. & eclipse signature problems
I'll start a review anyways.

**Comment by neubs-bsi** [comment]
@hanna-modica Conflicts not resolved yet. And needs rebase

File: assembly/cli/src/site/markdown/index.md.vm [hunk 1]
```diff
@@ -7,7 +7,7 @@ using the CLI frontend.
 
 #[[##]]# Configuring
 
-${docNameCap} CLI reads the same pom.xml configuration file that the Maven plugin version does.
+${docNameCap} CLI reads a configuration file (toolConfiguration.xml) that has the same structure as the Maven plugin.
```
Comment (neubs-bsi) [inline assembly/cli/src/site/markdown/index.md.vm]: Does this include the whole pom.xml blabla of maven? Maybe make a full example what all of the pom file needs to be in the toolConfigurationFile.

Comment (hanna-modica) [reply assembly/cli/src/site/markdown/index.md.vm]: How it should look like is described in the Tool Configuration Readme (that's why I renamed it from pom.xml). But I could probably add a link to that readme to help the reader find it.

**Comment by hanna-modica** [comment]
> 
> 
> I would add a snippet that shows a full `toolConfiguration.xml`, since I belive a full pom file is not necessary. And maybe use that in the example projects as well, to test for that level of simplicity

The full toolConfiguration.xml is here: https://github.com/eclipse/antenna/blob/e3d4ac8e3396bc8ba1515cfbcd3f6a8cf7a35a71/antenna-documentation/src/site/markdown/tool-configuration.md.vm
I also moved the hint, that the maven coordinates can be omitted, to the top of the example.

**Comment by neubs-bsi** [comment]
Follow up issue for ToolConfiguration Documentation has been created.

**Pull request merged**